### DISCUSSION TITLE
Feed contains a lot of linux kernel repo mirrors

### DISCUSSION THREAD

**Discussion by nicholasbishop** [community/community#38294] [Other Feature Feedback, Questions, & Ideas]
I have contributed the occasional commit to the Linux kernel, and because of that the Feed seems to think I've "contributed" to every repo that mirrors or forks the kernel (of which there are quite a few).

For example, I see this in my feed: 
![image](https://user-images.githubusercontent.com/1023208/200024876-0c2fb5f7-b11b-472e-93a0-e1f62f4be349.png)

In the `...` menu it says: "You're seeing this because you've contributed to xanmod/linux". Technically yes, but really no. It makes the new feed not very useful.

**Comment by david-engelmann** [comment]
Currently, this functionality isn't available

From github support ([support@github.com](mailto:support@github.com)).

"We can clear your entire public activity for you, but not individual events. Let me know if you'd like me to do that for you."
